package.json

```diff
@@ -22,6 +22,7 @@
     "vega": "5.4.0",
     "vega-lite": "3.3.0",
     "vega-embed": "4.2.0",
+    "@jupyter-widgets/base": "^1.1.10",
```

Comment:
Any reason not to require the latest version here?

Reply:
Not really. However, this version is anyway only used for type checking during typescript compilation. At runtime the version bundled with the widget extension is loaded. I will just update to the latest version and re-run the build steps.

Reply:
That's also the reason, why the js files are not changed, even after running yarn build.